dist/ng-obiba-mica.js

```diff
@@ -1114,7 +1112,292 @@ angular.module('obiba.mica.access')
+        chartAggregationName: '@',
+        chartEntityDto: '@',
+        chartOptionsName: '@'
+      },
+      templateUrl: 'graphics/views/charts-directive.html',
+      controller: 'GraphicChartsController'
+    };
+  }]);;/*
+ * Copyright (c) 2014 OBiBa. All rights reserved.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the GNU Public License v3.0.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+'use strict';
+
+angular.module('obiba.mica.graphics')
+
+  .controller('GraphicChartsController', [
+    '$rootScope',
+    '$scope',
+    '$filter',
+    'GraphicChartsConfig',
+    'GraphicChartsUtils',
+    'GraphicChartsData',
+    function ($rootScope,
+              $scope,
+              $filter,
+              GraphicChartsConfig,
+              GraphicChartsUtils,
+              GraphicChartsData) {
+      var ChartConfig = GraphicChartsConfig.getOptions();
+
+      GraphicChartsData.getData(function (StudiesData) {
+        if (StudiesData) {
+          $scope.ItemDataJSon = GraphicChartsUtils.getArrayByAggregation($scope.chartAggregationName, StudiesData[$scope.chartEntityDto], $scope.fieldTransformer);
+          $scope.ItemDataJSon.unshift(ChartConfig.ChartsOptions[$scope.chartOptionsName].header);
+          if ($scope.ItemDataJSon) {
+            $scope.chartObject = {};
+            $scope.chartObject.type = $scope.chartType;
+            $scope.chartObject.data = $scope.ItemDataJSon;
+            $scope.chartObject.options = {backgroundColor: {fill: 'transparent'}};
+            angular.extend($scope.chartObject.options, ChartConfig.ChartsOptions[$scope.chartOptionsName].options);
+            $scope.chartObject.options.title =  ChartConfig.ChartsOptions[$scope.chartOptionsName].options.title + ' (N=' + StudiesData.studyResultDto.totalHits + ')';
+
+          }
+        }
+      });
+
+    }]);
+;/*
+ * Copyright (c) 2014 OBiBa. All rights reserved.
+ *
+ * This program and the accompanying materials
+ * are made available under the terms of the GNU Public License v3.0.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+'use strict';
+
+angular.module('obiba.mica.graphics')
+  .factory('GraphicChartsGetData', ['$resource', 'ngObibaMicaUrl',
+    function ($resource, ngObibaMicaUrl) {
+      return $resource(ngObibaMicaUrl.getUrl('getStudiesStatistics'), {}, {
+        'get': {method: 'GET', errorHandler: true}
+      });
+    }])
+  .service('GraphicChartsConfig', function () {
+    var factory = {
+      options: {
+        IdNetworks: 'NaN',
+        ChartsOptions: {
+          geoChartOptions: {
+            header: null,
+            type: 'GeoChart',
+            options: {
+              title: null
+            }
+          },
+          recruitmentResources: {
```

Comment:
The default/real options should be passed by the client code (Drupal, Admin) via the directive attribute